### PR TITLE
ref: `queue kill --force` update &

### PR DESCRIPTION
**Pull request by jorgeorpinel**
and a link fix

> Follow-up to https://github.com/iterative/dvc.org/pull/4183

### CODE REVIEW

File: content/docs/command-reference/queue/kill.md [hunk 1]
```diff
@@ -39,5 +39,5 @@ To kill all running experiments and also stop processing the queue, use
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
-- `-f`, `--force` - Forcefully and immediately kill running experiment queue
-  tasks.
+- `-f`, `--force` - forcefully and immediately interrupt the task (not
+  graceful).
```
Comment (jorgeorpinel) [inline content/docs/command-reference/queue/kill.md:43]: Quick Q. `queue stop --kill` is also "forced" right? I.e. not graceful cc @dberenbaum

**Review by karajan1001 — APPROVED** [review]

File: content/docs/command-reference/queue/kill.md (hunk 1 above)
Comment (dberenbaum) [reply content/docs/command-reference/queue/kill.md:43]: Good point @jorgeorpinel. @karajan1001 What do you think?

**Comment by jorgeorpinel** [comment]
_Leaked Secrets Scan_ preventing merge here too @rogermparent

File: content/docs/command-reference/queue/kill.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/queue/kill.md:43]: Feel free to make this an issue in the core repo if you agree (or I can, lmk). Thanks

**Comment by rogermparent** [comment]
> _Leaked Secrets Scan_ preventing merge here too @rogermparent

I actually don't know what this is, it just kind of appeared today. I'll look into it.

**Comment by rogermparent** [comment]
Just have to push an empty commit! There's a message in #engineering on Slack talking about this.

**Pull request merged**